sphinx/io.py

```diff
@@ -136,12 +123,12 @@ class SphinxI18nReader(SphinxBaseReader):
     Because the translated texts are partial and they don't have correct line numbers.
     """
 
-    transforms = [ApplySourceWorkaround, ExtraTranslatableNodes, CitationReferences,
-                  DefaultSubstitutions, MoveModuleTargets, HandleCodeBlocks,
-                  AutoNumbering, SortIds, RemoveTranslatableInline,
-                  FilterSystemMessages, RefOnlyBulletListTransform,
-                  UnreferencedFootnotesDetector, SphinxSmartQuotes, ManpageLink,
-                  SubstitutionDefinitionsRemover]
+    def get_transforms(self):
+        # type: () -> List[Type[Transform]]
+        unused = [PreserveTranslatableMessages, Locale, RemoveTranslatableInline,
+                  AutoIndexUpgrader, FigureAligner, SphinxDomains, DoctreeReadEvent,
+                  UIDTransform]
+        return self.app.registry.get_transforms() - unused
```

Comment:
Incompatible change: This does not contains transforms which super class returns.